modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/pages/__tests__/UsageOverview.jsx

```diff
@@ -74,6 +74,68 @@ describe('UsageOverview', () => {
 		expect(container.querySelector('.alert-warning')).toBeInTheDocument();
 	});
 
+	it('should render the total number of INDIVIDUALS since last anniversary', () => {
+		const mockProject = new Project(
+			data.mockProject(23, {
+				faroSubscription: fromJS(
+					data.mockSubscription({
+						individualsCountSinceLastAnniversary: 1000,
+						individualsStatus: SubscriptionStatuses.Ok
+					})
+				)
+			})
+		);
+
+		const {getByText} = render(
+			<WrappedComponent {...defaultProps} project={mockProject} />
+		);
+
+		expect(getByText('1,000')).toBeInTheDocument();
+	});
+
+	it('should render the percentage of INDIVIDUALS used in the plan', () => {
+		const mockProject = new Project(
+			data.mockProject(23, {
+				faroSubscription: fromJS(
+					data.mockSubscription({
+						individualsCountSinceLastAnniversary: 1000,
+						individualsStatus: SubscriptionStatuses.Ok
+					})
+				)
+			})
+		);
+
+		const {getByText} = render(
+			<WrappedComponent {...defaultProps} project={mockProject} />
+		);
+
+		expect(getByText('1% since July 8, 2018')).toBeInTheDocument();
+	});
```

Comment:
Hi @fmarcio, could you group these two unit test?

Reply:
@interaminense done!